redis_api/api/routes.py

```diff
@@ -43,20 +46,18 @@ def get_candidates(candidate_id, serve_fake=False):
     :return: data on all candidates or for one specific candidate
     :rtype: JSON
     """
-    if serve_fake:
-        return fake_data.get_candidates_shape()
     try:
-        response = r.execute_command("JSON.GET", "Candidates")
-        if not response:
-            return empty_response("Candidates")
-        cand_json = json.loads(response)
+        redis = RedisClient()
+        candidateShape = redis.getCandidateShape()
         if candidate_id is None:
-            return jsonify({"Candidates": cand_json})
+            # TODO: Sync up with Front-End for the spec
```

Comment:
I believe we should be returning candidate information given a specific candidate ID. So we could either do that directly in this file or in services.

Reply:
I heard "We don’t really need that right now, because we just fetch all the candidates at once." today from the discussion with Emily. We could return just a single candidate info. The problem is the candidate ID is in a format ballot;name;date. The date contains slashes "/". To be able to implement this, we need to think how ID naming works.

Reply:
Gotcha, that should be easily fixable by calling .replace on the string. I will also be coming up with another way of ID'ing entities, so tbh not that big of a deal.